### ISSUE TITLE
Documentation: No demonstration (screencast)

### ISSUE THREAD

**Issue by Chealer** [josh-berry/tab-stash#507]
I am intrigued by Tab Stash, but I do not understand precisely what it offers. It would be much easier to decide whether I should try it if there was some demonstration. It could be textual, but probably the most efficient medium would be a screencast of 1 or 2 minutes.

<hr>
<em>Readers: If you are also interested in seeing this feature be developed, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button.  You are welcome to leave comments and discuss the feature request, but "Me too!" comments are not counted by the voting system.</em>

**Comment by josh-berry** [comment]
There's already an open issue for making videos, but before I mark this as a duplicate, I'm curious: did you go through the screenshots on [the Firefox add-ons page](https://addons.mozilla.org/en-US/firefox/addon/tab-stash/)? Most of the major functionality should be shown in the screenshots there.

What questions do you have and/or what do you feel is missing from the add-ons page?

**Comment by Chealer** [comment]
> There's already an open issue for making videos, but before I mark this as a duplicate, I'm curious: did you go through the screenshots on [the Firefox add-ons page](https://addons.mozilla.org/en-US/firefox/addon/tab-stash/)?

Yes, I did.

> Most of the major functionality should be shown in the screenshots there.

It may very well be; the problem may just be that it's unclear. In particular, it's unclear what relationship there is between the screenshots... is there a chronology?

> What questions do you have and/or what do you feel is missing from the add-ons page?

So, I suffer from tab hoarding and am looking for ways to keep my Firefox profile usable and performant which doesn't involve tab sacrifice. The "Clear your tabs, clear your mind." motto speaks to me. I understand Tab Stash's basic purpose, but I fail to precisely understand how it works. Specifically, what does "stashing a tab" mean? What are "unstashed tabs"?

**Comment by josh-berry** [comment]
> > Most of the major functionality should be shown in the screenshots there.
> 
> It may very well be; the problem may just be that it's unclear. In particular, it's unclear what relationship there is between the screenshots... is there a chronology?

There is not, mainly because it was hard to show all the things I wanted to in the screenshots. But this is really helpful feedback, I'll consider re-doing them to make it more clear exactly what happens and how to use it.

> > What questions do you have and/or what do you feel is missing from the add-ons page?
> 
> So, I suffer from tab hoarding and am looking for ways to keep my Firefox profile usable and performant which doesn't involve tab sacrifice. The "Clear your tabs, clear your mind." motto speaks to me. I understand Tab Stash's basic purpose, but I fail to precisely understand how it works. Specifically, what does "stashing a tab" mean? What are "unstashed tabs"?

Alright, so when you stash a tab:

- It gets saved into a "Tab Stash" folder in your bookmarks
- It gets hidden or closed, depending on your preferences (by default, it's hidden but remains open in the background)

When you un-stash a tab:

- It's re-opened (or un-hidden, if previously hidden)
- It is optionally (depending on which button you click) also removed from your bookmarks

"Unstashed tabs" are therefore tabs that are not saved in your Tab Stash bookmarks.

That's pretty much it! It's intended to be a simple, convenient way to move individual tabs and groups of tabs between your bookmarks and your browser, and optionally keep open (but out of sight) things you don't want to see right now but might want to come back to later.

Did that help to explain things better? Do you have any more questions?

**Comment by Chealer** [comment]
> > [...]
> 
> Did that help to explain things better?

Absolutely! Thank you very much Josh, that sounds so interesting that I just installed and am starting to test it now :-)

> Do you have any more questions?

Now that I have it, no, but thanks. For the record and to be perfectly clear, all tabs which are not in the stash are "unstashed" (whether they previously were or not).

**Comment by josh-berry** [comment]
Thanks for the suggestions! The point of the text and screenshots on the add-ons page is to draw people in and show them a little bit of what Tab Stash is about, enough to help them decide whether to try it or not. 

Trying to give detailed explanations right away would, I think, very quickly become overwhelming. So my goal is to keep the intro text really really simple, use the screenshots to illustrate the key points, and convince people to play around with it to see if it's right for them.

That said, I still definitely agree with you that having a more detailed walkthrough and explanation would be useful, I'm just not sure the add-on listing is the right place for that.

**Comment by unbob** [comment]
Thanks much for the discussion - very informative!

I've just thrown in the towel on the OneTab extension after using it for several years. It's concept is great but much too easy for the user to "screw-up" and lose tabs and windows - then spend hours trying to recover. I'm done with that!

So I will try TS and hope it meet my needs. Hopefully, it's safer to use and harder for the user to make a mistake that is catastrophic. Thanks for your time & effort creating TS!
